crates/frontend/src/types.rs

```diff
@@ -122,10 +126,71 @@ pub struct ExperimentsResponse {
 pub struct StatusTypes(pub Vec<ExperimentStatusType>);
 
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct ExpListFilters {
+pub struct ExperimentListFilters {
     pub status: Option<StatusTypes>,
     pub from_date: Option<DateTime<Utc>>,
     pub to_date: Option<DateTime<Utc>>,
+    pub all: Option<bool>,
+    pub page: Option<i64>,
+    pub count: Option<i64>,
```

Comment:
`PaginationParams` can be used for this, the filters can be separate from pagination parameters.

Reply:
Sure, will make that change